### ISSUE TITLE
webUI tests do not run with latest docker-selenium 3.141.59-palladium

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#35444]
When using the release tag https://github.com/SeleniumHQ/docker-selenium/releases/tag/3.141.59-palladium the webUI tests timeout trying to access the login screen.

Others are also having problems, see https://github.com/SeleniumHQ/docker-selenium/issues/919

Example CI run is https://github.com/owncloud/core/pull/35441 https://drone.owncloud.com/owncloud/core/18073/681

**Comment by phil-davis** [comment]
For now, we are reverting drone CI to use the previous `selenium/standalone-chrome-debug:3.141.59-oxygen` which has chrome 74.
`master` #35442 
`stable10` #35443

**Comment by individual-it** [comment]
`phoenix` https://github.com/owncloud/phoenix/pull/1260

**Comment by phil-davis** [comment]
This is likely related to https://bugs.chromium.org/p/chromedriver/issues/detail?id=2536
"Make standards mode (goog:chromeOptions.w3c:true) the default"

**Comment by phil-davis** [comment]
It is not proving easy to use chrome+chromedriver 75 with `w3c:false`
See blog https://medium.com/@alex.designworks/chromedriver-75-enforces-w3c-standard-breaking-behat-tests-460cad435545

**Comment by phil-davis** [comment]
For the moment this is blocked - there is no obvious solution for running Behat-Mink-selenium with chrome+chromedriver 75
The fallback to chrome 74 has been implemented everywhere and is OK "for the near future"

**Comment by phil-davis** [comment]
Added QA-p2 label because this needs to be followed up reasonably often to see if anything can be done.

**Comment by saw-jan** [comment]
Couldn't make webUI tests pass with `selenium/standalone-chrome-debug:latest`.
But UI tests were passing locally with `latest` i.e. version `selenium/standalone-chrome-debug:3.141.59` image.

Also tried creating new Behat-Mink project with some UI test:
Config:
  - behat `^3.8`
  - mink `^1.8`
  - mink-extension `^2.3`
  - mink-selenium2-driver `^1.4`
 
and
  - selenium/standalone-chrome-debug:`3.141.59` (chrome: 90, chromeDriver: 90)

UI tests were passing with these setups too.

Conclusion:
**Couldn't find a promising solution to use `selenium/standalone-chrome-debug:latest` and make all UI tests passing in drone.**

**Comment by DeepDiver1975** [comment]
**Classification: analysis-only** (valid bug, working-as-intended for now — no fix actionable here)

🤖 Automated issue analysis by Claude Code review agent.

I checked this against current `master` (HEAD `7da5538`, June 2026). Notably, this is **not yet obsolete** — the situation it describes is still in effect:

- The CI has since migrated from Drone to GitHub Actions (`.github/workflows/acceptance.yml`, June 2026). The WebUI/Selenium job there **still pins the 2019 fallback image** `selenium/standalone-chrome-debug:3.141.59-oxygen` (chrome 74) — `acceptance.yml:327` — i.e. exactly the workaround @phil-davis put in place at the time. The legacy `.drone.star` pins the same image (`.drone.star:24`).
- The root problem reported here — that newer docker-selenium / chrome+chromedriver (≥75, w3c default) breaks the Behat-Mink-Selenium2 stack — was never resolved upstream in a way that let this project move off the frozen image. So the title ("webUI tests do not run with latest docker-selenium 3.141.59-...") is still literally accurate against newer images.
- That said, WebUI acceptance tests are now gated behind a `do-webui-tests` input that **defaults to `false`** (`acceptance.yml:39`, after #41607 "run only API and CLI smoke tests by default"). So in practice the frozen-chrome-74 setup is no longer on the hot path of default CI.

**Conclusion:** this remains a real, still-open CI/test-infra limitation, but it is effectively working-as-intended (intentional pin to a known-good image) with no fix path within this project — the blocker is in the Behat-Mink-Selenium2 ↔ chromedriver-w3c interaction, not ownCloud code. There is nothing to fix on `owncloud/core` here today, so I'm leaving it open as-is rather than closing.

If anyone does attempt to move off the `3.141.59-oxygen` image again, this issue is the right place to track it.

**Comment by phil-davis** [comment]
Assigned myself so that I find this issue.
I think there was another issue about this, so when I find it, I will close one of them.

**Comment by dj4oC** [comment]
Before this can be acted on, the following needs to be resolved:

- Confirmed still accurate: current CI (`.github/workflows/acceptance.yml`) still pins the 2019 fallback image `selenium/standalone-chrome-debug:3.141.59-oxygen` (Chrome 74) for WebUI acceptance tests — the underlying Behat-Mink-Selenium2 ↔ chromedriver-w3c incompatibility with newer docker-selenium images was never resolved upstream.
- This is blocked on an external dependency (the Behat-Mink-Selenium2-driver stack's lack of W3C WebDriver support against modern chromedriver), not something fixable in this repo's own code — any real fix would mean migrating the acceptance-test driver stack, which is a larger test-infra project, not a code patch.
- Since #41607, WebUI acceptance tests are gated behind a `do-webui-tests` input defaulting to `false`, so this is no longer on the default CI hot path. Given that mitigation is already in place, is this still worth tracking as an open, actively-followed issue (per the existing `QA:p2` label), or should it be downgraded/closed until someone picks up a driver-stack migration?

🤖 Automated by Claude Code

---
_Generated by [Claude Code](https://claude.ai/code)_
